src/quadratic_model.jl

```diff
@@ -136,6 +136,7 @@ function complete_quadratic_model!(Q::QmIGE, c, box::Box{T,N}, f::Function, x0,
             elseif xcur >= u
                 u, xcur = xcur, (l + xcur)/2
             end
+            abs(u-l) < epswidth((l, u)) && return Q #FIXME?
```

Comment:
What about the case `u == l == 0`?

Reply:
I thought we would still want to abort and return Q in that case.  This does evaluate to true:
```julia
using QuadDIRECT
0.0 - 0.0 < QuadDIRECT.epswidth((0.0, 0.0))
```
Do we want that case to evaluate to false?

Reply:
Now that I think about it maybe that reflects a bug in `epswidth`?  My guess was that `epswidth` is meant to return something proportional to the distance between `l` or `u` and the closest representable number *between* `l` and `u`.  If that's the case then `epswidth(0.0, 0.0)` would be undefined, or zero.

Reply:
Right, if `u == l == 0` then `epswidth` returns 0. But then `0 < 0` is false, so it doesn't abort (and should).

Reply:
But I'm saying that `epswidth` doesn't return 0 in that case.  If you copy-paste my code above you'll see it's true.

Reply:
(for me `epswidth(0.0,0.0)` is 5.0e-323)

Reply:
Oh! Sorry, my mistake! Carry on! :smile:

Reply:
But I agree that intuitively `epswidth` should return zero in that case...I wonder if that's a source of some of these eps-related bugs.  Of course if I change the `epswidth` function there could be other repercussions.  Up to you whether to merge as-is

Reply:
It turns out `eps(0.0) = 5.0e-324` so I think that's consistent.